docs/release-1.3/README.md

```diff
@@ -0,0 +1,309 @@
+
+    Sample of a secret created in a GitOps environment:
+
+    ```console
+    $> oc get secret -n rhdh-operator -o yaml backstage-backend-auth-secret
+    apiVersion: v1
+    data:
+      ARGOCD_PASSWORD: ...
+      ARGOCD_URL: ...
+      ARGOCD_USERNAME: ...
+      BACKEND_SECRET: ...
+      GITHUB_TOKEN: ...
+      K8S_CLUSTER_TOKEN: ...
+      K8S_CLUSTER_URL: ...
+    kind: Secret
+    metadata:
+      creationTimestamp: "2024-05-07T22:22:59Z"
+      name: backstage-backend-auth-secret
+      namespace: rhdh-operator
+      resourceVersion: "4402773"
+      uid: 2042e741-346e-4f0e-9d15-1b5492bb9916
+    type: Opaque
+    ```
+1.  Use the following manifest to install the operator in an OCP cluster:
+
+    ```yaml
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: orchestrator-operator
+      namespace: openshift-operators
+    spec:
+      channel: alpha
+      installPlanApproval: Automatic
+      name: orchestrator-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+    ```
+
+1.  Run the following commands to determine when the installation is completed:
+
+    ```console
+    wget https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/main/hack/wait_for_operator_installed.sh -O /tmp/wait_for_operator_installed.sh && chmod u+x /tmp/wait_for_operator_installed.sh && /tmp/wait_for_operator_installed.sh
+    ```
+
+    During the installation process, Kubernetes cronjobs are created by the operator to monitor the lifecycle of the CRs managed by the operator: RHDH operator, OpenShift Serverless operator and OpenShift Serverless Logic operator. When deleting one of the previously mentioned CRs, a job is triggered that ensures the CR is removed before the operator is.
+    In case of any failure at this stage, these jobs remain active, facilitating administrators in retrieving detailed diagnostic information to identify and address the cause of the failure.
+
+    > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the orchestrator resource values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the orchestrator resource is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
+
+1. Apply the Orchestrator custom resource (CR) on the cluster to create an instance of RHDH and resources of OpenShift Serverless Operator and OpenShift Serverless Operator Logic.
+   Make any changes to the [CR](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/config/samples/_v1alpha1_orchestrator.yaml) before applying it, or test the default Orchestrator CR:
+    ```console
+    oc apply -n orchestrator -f https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/main/config/samples/_v1alpha1_orchestrator.yaml
```

Comment:
Should be `https://github.com/parodos-dev/orchestrator-helm-operator/blob/release-1.3/config/samples/_v1alpha1_orchestrator.yaml`
and 
`https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/release-1.3/config/samples/_v1alpha1_orchestrator.yaml`